darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1300,6 +1300,36 @@ def save_model(self, path: str) -> None:
         # TODO: the parameters are saved twice currently, once with complete
         # object, and once with PTL checkpointing.
 
+        if path is None:
+            # default path
+            path = f"{type(self).__name__}_{datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}.pt"
```

Comment:
Could we maybe extract this out in a `ForecastingModel.get_default_save_path()` method?
This would avoid having this default convention written twice (in ForecastingModel and TorchForecastingModel).

Reply:
Sounds like a good idea. I didn't do it because of the different filename extensions (.pt and .pkl) but now I'll just add the filename extensions separately and use `ForecastingModel.get_default_save_path()` for the main part.